### ISSUE TITLE
Accordion: Fix UI mismatches 

### ISSUE THREAD

**Issue by franciscrispin** [glints-dev/glints-aries#183]
Glints UI: https://app.zeplin.io/project/5ba343bf0c399c33db483653/screen/5ba358620049d012fcb8b234 
Glints Aries Status: https://docs.google.com/spreadsheets/d/1O9WY_x9MmzRIbskPE-2_TsKgOz3N7C61iEqcx6zWln8/

1. Identify the UI Mismatches
2. Fix the UI Mismatches
3. Implement Unit Tests for the component

- In addition to the design on Glints UI, the icon position should be customizable to be on the left or right

**Comment by franciscrispin** [comment]
Minor UI mismatches remaining: 
- animate the change from plus to minus
- start the divider only after after the icon

**Comment by westwood846** [comment]
Closing, can make new issue when designs are available.
